src/5e-SRD-Monsters.json

```diff
@@ -905,39 +905,43 @@
           "dice": "1d6",
           "min_value": 5
         },
-        "attacks": [
-          {
-            "name": "Fire Breath",
-            "dc": {
-              "dc_type": {
-                "name": "DEX",
-                "url": "/api/ability-scores/dex"
+        "attack_options": {
+          "choose": 1,
+          "type": "attacks",
```

Comment:
Should the `type` of these choices be "attack" (singular)? Elsewhere, the choice structure's `type` attribute is in the singular.

Reply:
I think I must have gotten confused. I'll fix it.

Reply:
Fixed!